lightclient-circuits/src/witness/rotation.rs

```diff
@@ -59,17 +63,29 @@ impl<S: Spec> Default for CommitteeUpdateArgs<S> {
             &sync_committee_branch,
             S::SYNC_COMMITTEE_PUBKEYS_ROOT_INDEX,
         );
+        let finalized_header = BeaconBlockHeader {
+            state_root: state_root.as_slice().try_into().unwrap(),
+            ..Default::default()
+        };
+
+        let (finalized_header_multiproof, finalized_header_helper_indices) =
+            beacon_header_multiproof_and_helper_indices(
+                &mut finalized_header.clone(),
+                &[S::HEADER_STATE_ROOT_INDEX],
+            );
 
         Self {
             pubkeys_compressed: iter::repeat(dummy_x_bytes)
                 .take(S::SYNC_COMMITTEE_SIZE)
                 .collect_vec(),
             sync_committee_branch,
-            finalized_header: BeaconBlockHeader {
-                state_root: state_root.as_slice().try_into().unwrap(),
-                ..Default::default()
-            },
+            finalized_header,
             _spec: PhantomData,
+            finalized_header_multiproof: finalized_header_multiproof
+                .into_iter()
+                .map(|n| n.as_ref().to_vec())
```

Comment:
why not return Vec<Vec<u8>> in `beacon_header_multiproof_and_helper_indices`

Reply:
lol yeah ok agree